docs/development/qubes_staging.rst

```diff
@@ -245,8 +239,9 @@ to set up the development environment.
 
 Once finished, build the Debian packages for installation on the staging VMs:
 
-- **Xenial:** use the command ``make build-debs``
-- **Focal:** use the command ``make build-debs-focal``
+.. code::  sh
+
+    make build-debs-focal
```

Comment:
That Makefile target does not exist, it's just build-debs

Reply:
Thank you, fixed!